argoverse/evaluation/stereo/utils.py

```diff
@@ -128,7 +128,8 @@ def accumulate_stereo_metrics(abs_error_thresholds: List[int]) -> pd.DataFrame:
     return pd.DataFrame([[0] * num_fields], columns=columns)
 
 
-def interpolate_disparity(disp: np.ndarray) -> np.ndarray:
+@njit(nogil=True)  # type: ignore
+def interpolate_disparity(disparity: np.ndarray) -> np.ndarray:
```

Comment:
This is impressive the numba code can run so fast. @jhonykaesemodel could we add a few tiny unit tests for this function?

Reply:
Sure, I'll think about some toy tests to add in.